w_common/lib/src/browser/disposable_browser.dart

```diff
@@ -310,15 +310,15 @@ class Disposable implements disposable_common.Disposable {
   /// method on the window singleton to remove the listener. At this point
   /// the only way to remove the listener is to use the [dispose] method.
   void subscribeToWindowEvent(String event, EventListener callback,
-      {bool useCapture, EventTarget windowObject}) {
+      {bool? useCapture, EventTarget? windowObject}) {
     if (windowObject == null) {
       windowObject = window;
     }
     _subscribeToEvent(windowObject, event, callback, useCapture);
   }
 
   void _subscribeToEvent(EventTarget eventTarget, String event,
-      EventListener callback, bool useCapture) {
```

Comment:
I think `useCapture` should be non-nullable. It's really weird for a boolean to be nullable.